.github/workflows/dependency-review.yml

```
@@ -0,0 +1,21 @@
+name: "Dependency review"
+on:
+  [pull_request, push]:
```

actionlint (1.7.9)

expected scalar node for string value but found sequence node with "!!seq" tag

(syntax-check)